module/applications/item/item-directory.mjs

```diff
@@ -24,4 +25,13 @@ export default class ItemDirectory5e extends ItemDirectory {
     // Let parent method perform sorting
     super._handleDroppedEntry(target, item.toDragData());
   }
+
+  /* -------------------------------------------- */
+
+  /** @override */
+  async _onClickEntryName(event) {
+    const { entryId } = event.target.closest("[data-entry-id]")?.dataset ?? {};
+    const item = this.collection.get(entryId);
+    item?.sheet.render(true, { mode: ItemSheet5e2.MODES.EDIT });
```

Comment:
For sidebar I'm still a bit unsure what the preferred mode would be. Maybe it should only force a mode if the sheet hasn't been opened yet?

Reply:
I think we should only ever apply the mode on first render. If you've previously had the sheet open from anywhere it would be nice if it retained the previous mode on re-open especially since the View button isn't displayed in the actor inventory unless you don't have edit permission, so if you want to view an item in play mode you have to change the mode every time you open it (or we could make the View button always visible so players always have control over what mode it opens with).

Reply:
That does seem like it would offer the least friction. I will make the change.

Reply:
I have made this change now, though I think adding a separate view context menu option could also work. These behaviours are mutually exclusive so I think it might come down to user preference.

I think there are still issues with this technique. Presumably if an item is enchanted to be magical, for example, then you should be able to toggle attunement in Play mode, but that field will be disabled unless in Edit mode.

I think a more complete solution will involve identifying areas, like attunement, that are the target of various enchantments, and moving those out of this form-based editing, and into the sheet 'proper'. Attunement could be a header button, like equipped and identified, for example.

If enchantments get particularly sophisticated, then perhaps we need an entirely bespoke 'details' tab that is not a form at all when in Play mode, and only show the form in Edit mode.

For now, though, this is probably good enough to unblock some issues while we gather feedback.